### ISSUE TITLE
Revised glyph order data

### ISSUE THREAD

**Issue by tphinney** [tphinney/science-gothic#27]
I'm in the process of revising the glyph ordering in the spreadsheet, which is the source of the encoding file, etc. Things like putting all the western diacritics together, putting copyright, registered and trademark together, putting the one superior next to the 2 and 3 superior, etc. It should be easier to find things afterwards.

**Comment by tphinney** [comment]
This exercise is also uncovering at least one bug:
- prime and double prime glyphs appeared twice in the list (fixed for next version)

**Comment by tphinney** [comment]
Also, trademark appeared twice.

Currently this is about 2/3 done. Working on it when I get tired of other tasks.

**Comment by tphinney** [comment]
Done! Found and fixed a few more glitches, including a small number (~ 8?) of incorrect unicode values (and uniXXXX names). Igor will generate a new ".enc" file.

**Comment by tphinney** [comment]
What's new in the latest .enc file? A preliminary version was released about a week ago. (after the trademark fix mentioned above). Here's what has happened since then:

1. So, MOSTLY there is a LOT more re-ordering. The previous encoding was unfinished as far as re-ordering goes. This time I put in another 3+ hours and finished my sorting of some blocks of letters with diacritics. I also found various other clumps of related glyphs that are more usefully grouped together. (For example: hyphen, en dash, em dash, and their cap variants are all grouped together now.)

1. A very small number of glyphs occurred twice! There were two or four that I found manually, and five that Igor's process found and eliminated after sorting the list. So that means five are still in the spreadsheet and must be found.

1. The .enc file sorts in the "OpenType" group in FontLab. This is result of a single line in header. This way people with a zillion .enc files do not have to scroll several screens to get to it.

1. For now, I removed the 34 glyphs listed here: https://github.com/tphinney/science-gothic/issues/59 — these were small cap glyphs that have no matching cap or lowercase. They could come back later.
